package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
```

Comment:
@dapetcu21 I'd like to get both android and ios on the latest version. We just released a big [bug fix release](http://status.mapbox.com/incidents/59f09r86vhyt) and I'd like to make sure we're on this. Theoretically, there should be no breaking changes since this branch is already on 3.x.